symengine/cwrapper.cpp

```diff
@@ -41,6 +41,8 @@ extern "C" {
 
 struct CRCPBasic {
     SymEngine::RCP<const SymEngine::Basic> m;
+    CRCPBasic() {};
```

Comment:
The error was because of a silly mistake that I made in the tests. I fixed them now.